internal/processing/istio/authorization_policy_processor_test.go

```diff
@@ -194,4 +196,172 @@ var _ = Describe("Authorization Policy Processor", func() {
 		Expect(ap.OwnerReferences[0].Name).To(Equal(ApiName))
 		Expect(ap.OwnerReferences[0].UID).To(Equal(ApiUID))
 	})
+
+	When("single handler only", func() {
+		It("should not create AP for allow", func() {
```

Comment:
We could add a separate test to verify that APs created for jwt handlers contain a Spec.Rules.From.